rtcp.c

```diff
@@ -133,6 +133,47 @@ static void janus_rtcp_incoming_sr(janus_rtcp_context *ctx, janus_rtcp_sr *sr) {
 	ctx->lsr = (ntp >> 16);
 }
 
+/* Link quality estimate filter coefficient */
+#define LINK_QUALITY_FILTER_K 3.0
+
+static double janus_rtcp_link_quality_filter(double last, double in) {
+	if (last == 0) {
+		return in;
+	}
+	else {
```

Comment:
} else {

As a code convention, we put the `else` on the same line as the closing backet.